src/components/dialog/dialog-container.ts

```diff
@@ -0,0 +1,70 @@
+import {Component, ComponentRef, ViewChild, AfterViewInit} from '@angular/core';
+import {BasePortalHost, ComponentPortal, TemplatePortal} from '../../core/portal/portal';
```

Comment:
I thought it be worth to add some checks for that. Also with auto-suggestion. See #764 
This would probably help avoiding those common mistakes.